src/autoscaler/api/publicapiserver/public_api_server_test.go

```diff
@@ -42,24 +46,24 @@ var _ = Describe("PublicApiServer", func() {
 	)
 
 	BeforeEach(func() {
-		scalingHistoryEntry := []scalinghistory.HistoryEntry{
+		scalingHistoryEntry := []internalscalinghistory.HistoryEntry{
```

Comment:
Q. internationScalinghistory name is becuase that we already have scalinghistory object or does it also offers some additional functionality ?